src/core/src/pass/graph_rewrite.cpp

```diff
@@ -60,6 +64,9 @@ PerfCounters& perf_counters_graph_rewrite() {
 }  // namespace
 }  // namespace pass
 }  // namespace ov
+#if defined __GNUC__
```

Comment:
why do we need it here? if for CC build, we can use https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/CMakeLists.txt#L36-L41